src/go/rdctl/cmd/root.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright © 2022 SUSE LLC
+
+*/
+
+// Package cmd is the main package for this CLI
+package cmd
+
+import (
+  "encoding/json"
+  "fmt"
+  "io/ioutil"
+  "log"
+  "net/http"
+  "os"
+  "path/filepath"
+  "strconv"
+
+  "github.com/spf13/cobra"
+)
+
+var (
+  // Used for flags
+  configPath  string
+  user string
+  port string
+  password string
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "rdctl",
+	Short: "A CLI for Rancher Desktop",
+	Long: `rdctl can be used to drive Rancher Desktop in headless mode.
+Supported commands include getting the current preferences,
```

Comment:
This list will get out of date fast; better to not have it at all.

Reply:
Right, replaced with more generic language.